lapack/testlapack/locallapack.go

```diff
@@ -0,0 +1,162 @@
+// Copyright ©2020 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package testlapack
+
+import (
+	"math"
+
+	"gonum.org/v1/gonum/blas"
+	"gonum.org/v1/gonum/lapack"
+)
+
+func dlagtm(trans blas.Transpose, m, n int, alpha float64, dl, d, du []float64, b []float64, ldb int, beta float64, c []float64, ldc int) {
```

Comment:
Note that this is copied from `gonum.Implementation`'s method to avoid cyclic dependencies.

Maybe we should do the testing for lapack/gonum in gonum_test? (#future)

Reply:
Would there actually be cyclic dependencies? I'm not sure now. I started making copies of `Dlan*` functions because some of them are not in lapacke. If not, couldn't we call into `gonum.Implementation` directly without putting them into the impl parameter's interface? If we keep these copies, maybe move them into `testlapack.internal`?

Reply:
Of course there are currently cyclic dependencies. Now I fully understand your comment. Yes, #future.